### ISSUE TITLE
Failing hive engine-withdrawals tests

### ISSUE THREAD

**Issue by emhane** [paradigmxyz/reth#8304]
#### Withdrawals Fork
- [ ]  test=2 name="Withdrawals Fork On Genesis (Paris) (reth)”
- [ ]  test=3 name="Withdrawals Fork on Block 1 (Paris) (reth)”
- [ ]  test=4 name="Withdrawals Fork on Block 2 (Paris) (reth)”
- [ ]  test=5 name="Withdrawals Fork on Block 3 (Paris) (reth)”

- [ ]  test=20 name="Withdrawals Fork on Block 1 - 8 Block Re-Org NewPayload (Paris) (reth)”
- [ ]  test=22 name="Withdrawals Fork on Block 8 - 10 Block Re-Org NewPayload (Paris) (reth)”
- [ ]  test=21 name="Withdrawals Fork on Block 1 - 8 Block Re-Org, Sync (Paris) (reth)”
- [ ]  test=23 name="Withdrawals Fork on Block 8 - 10 Block Re-Org Sync (Paris) (reth)”
- [ ]  test=24 name="Withdrawals Fork on Canonical Block 8 / Side Block 7 - 10 Block Re-Org (Paris) (reth)”
- [ ]  test=26 name="Withdrawals Fork on Canonical Block 8 / Side Block 9 - 10 Block Re-Org (Paris) (reth)”
- [ ]  test=25 name="Withdrawals Fork on Canonical Block 8 / Side Block 7 - 10 Block Re-Org Sync (Paris) (reth)”
- [ ]  test=27 name="Withdrawals Fork on Canonical Block 8 / Side Block 9 - 10 Block Re-Org Sync (Paris) (reth)”

#### Withrdawals to X account(s)

- [ ]  test=6 name="Withdraw to a single account (Paris) (reth)”
- [ ]  test=7 name="Withdraw to two accounts (Paris) (reth)”
- [ ]  test=8 name="Withdraw many accounts (Paris) (reth)”

#### Withdraw zero amount

- [ ]  test=9 name="Withdraw zero amount (Paris) (reth)”

#### Empty withdrawals

- [ ]  test=10 name="Empty Withdrawals (Paris) (reth)”

#### Corrupted Block Hash Payload (INVALID)

- [ ]  test=11 name="Corrupted Block Hash Payload (INVALID) (Paris) (reth)”

**Comment by emhane** [comment]
is this issue closed @fgimenez ?

**Comment by fgimenez** [comment]
> is this issue closed [@fgimenez](https://github.com/fgimenez) ?

nope, we still have these tests as expected failures https://github.com/paradigmxyz/reth/blob/main/.github/assets/hive/expected_failures.yaml#L24-L42

**Comment by jenpaff** [comment]
tracked in expect_failures https://github.com/paradigmxyz/reth/blob/eaa45abb989646fad41b38a161e2f948ec36df7a/.github/assets/hive/expected_failures.yaml#L20
